pyvista/plotting/widgets.py

```diff
@@ -466,6 +467,12 @@ def add_plane_widget(
             .. versionchanged:: 0.38.0
                Now accepts either strings and ``vtk.vtkCommand.EventIds``.
 
```

Comment:
We should also tell the user they can use a float value.  I'm okay if we also accept a Boolean as this converts correctly to the behavior.

Reply:
Thanks a lot! resolved in 48405a1d99333525fd6fed7c7cc399aded6d2761 1aceb6f7eed7e415650c267ba5ec881e1905af04 02ca44414de704c50e070e5c2adff77e399eafef .